e2e-tests/insurance/cypress/e2e/journeys/application-submission/single-policy-type/buyer-conditions/submit-an-application-single-policy-type-buyer-no-financial-accounts.spec.js

```diff
@@ -1,9 +1,9 @@
-context('Insurance - submit an application - Single policy type, no broker - As an Exporter, I want to submit my completed credit insurance application with a value greater than 500000, So that UKEF can process and make a decision on my application', () => {
+context('Insurance - submit an application - Single policy type, fully populated buyer', () => {
```

Comment:
add no financial accounts to context?

Reply:
Good shout, updated for single and multiple policy tests